superx/routes/signup.py

```diff
@@ -41,8 +44,8 @@ def validate_email(self, email):
         import requests
 
         response = requests.get(
-        "https://isitarealemail.com/api/email/validate",
-        params = {'email': email.data})
+            "https://isitarealemail.com/api/email/validate",
```

Comment:
Did you decide that this was better than using the wtforms email validator

Reply:
For now, we want to use any kind of validation that will work. Because the wtforms email validator doesn't work on the vagrant machine we use this kind of validation. I belive that @aryehlev working on this part now.